tasks/main.yml

```diff
@@ -4,7 +4,7 @@
 
 - name: include scl related overrides
   include_vars: "{{ ansible_distribution }}-scl.yml"
-  when: install_from_src != true
+  when: not install_from_src
```

Comment:
I've never had good experience with using `not`.  Maybe, now that I think about it, using `| bool`? Either way this include [get's skipped](https://travis-ci.com/OSC/ood-ansible/builds/144929225#L1740) and the [one below](https://travis-ci.com/OSC/ood-ansible/builds/144929225#L2028) are being skipped and that's why tests are failing.

Reply:
This is happening because you're passing the value for this variable in command line, and:
![image](https://user-images.githubusercontent.com/3178490/72717815-aa6d0680-3b6c-11ea-8fa7-f0b818491c14.png)
https://docs.ansible.com/ansible/2.4/playbooks_variables.html#passing-variables-on-the-command-line
https://github.com/ansible/ansible/issues/17094#issuecomment-240793138

I think instead of using `| bool` filter we should pass the CLI variable as a json: `-e '{ "install_from_src": false }'`